src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -571,6 +573,7 @@ protected BoundBlock BindLambdaBody(LambdaSymbol lambdaSymbol, Binder lambdaBody
                 Interlocked.Increment(ref data.LambdaBindingCount);
             }
 
+            Binder.RecordLambdaBinding(UnboundLambda.Syntax);
```

Comment:
>UnboundLambda.Syntax [](http://example.com/codeflow?start=39&length=20)

If the syntax is readily available from ```UnboundLambda```, do we really need an extra field for the syntax?

Reply:
Good catch, thanks. The field was left over from an earlier version of this change.